src/vellum/workflows/workflows/base.py

```diff
@@ -539,7 +539,7 @@ def stream(
             subworkflows or nodes that utilizes threads.
         """
 
-        should_yield = event_filter or workflow_event_filter
+        should_yield = event_filter or root_workflow_event_filter
```

Comment:
No. We cannot change the default behavior here as it would be a breaking change. Please revert this assumption and never make it again unless explicitly asked